client/src/api.js

```diff
@@ -0,0 +1,547 @@
+// ******************** GET DATA APP getCurrentProfile ********************
+
+export const getCurrentProfile = async () => {
+
+  let response;
+
+  try {
+    response = await fetch(
+      `${window.config.baseUrl}/users/current/profile`, {
+        headers: {
+          Authorization: 1
```

Comment:
Drop the `Authorization` header from all `fetch` calls, it is not needed anymore.

Reply:
should i remove the headers in every fetch entirely if empty? or should be there an empty object  // headers: {},

Reply:
Entirely.